pkg/app/app.go

```diff
@@ -25,16 +25,21 @@ var (
 	// routed.
 	NotFound UI = &notFound{}
 
-	routes = make(map[string]UI)
+	routes router
 	uiChan = make(chan func(), 256)
 )
 
 // EventHandler represents a function that can handle HTML events.
 type EventHandler func(src Value, e Event)
 
 // Route binds the requested path to the given UI node.
-func Route(path string, n UI) {
-	routes[path] = n
+func Route(path string, node UI) {
+	routes.route(path, node)
+}
+
+// RouteWithRegexp binds the regular expression pattern to the given UI node.
```

Comment:
Maybe we should add that it is following the Go standard regexp format.

Reply:
done